limited/accountant

```diff
@@ -0,0 +1,6 @@
+**Accountant** | Townsfolk Miscellaneous | Limited
+__Basics__
+The Accountant is told how many wolves are in each half of the town.
```

Comment:
```suggestion
The Accountant is told how many werewolves are in each half of the town.
```
I'm gonna guess you mean werewolves (as in members of the werewolf team) instead of wolves (as in players with the wolf role)